packages/@uppy/core/src/index.js

```diff
@@ -175,13 +162,6 @@ class Uppy {
     this.updateOnlineStatus = this.updateOnlineStatus.bind(this)
     this.resetProgress = this.resetProgress.bind(this)
 
-    this.pauseAll = this.pauseAll.bind(this)
-    this.resumeAll = this.resumeAll.bind(this)
-    this.retryAll = this.retryAll.bind(this)
-    this.cancelAll = this.cancelAll.bind(this)
-    this.retryUpload = this.retryUpload.bind(this)
-    this.upload = this.upload.bind(this)
-
     this.emitter = ee()
     this.on = this.on.bind(this)
     this.off = this.off.bind(this)
```

Comment:
these could also be removed i think

Reply:
Done in https://github.com/transloadit/uppy/pull/3042, I wanted to avoid a git conflict.

Reply:
perfect 👍🏻